### DISCUSSION TITLE
Customizable Save Locations

### DISCUSSION THREAD

**Discussion by ranshaa05** [open-goal/jak-project#3520] [Ideas]
Having a feature that would allow the user to customize their savegame location could be a nice feature to have.

One potential use case is the creation of a cloud-based save system. Users could store their save files on a cloud service like OneDrive or Google Drive, enabling them to synchronize their saves across different devices.

Another potential application is if Jak X were to be released, users could place a Ratchet & Clank save file in the specified location to unlock Ratchet as a playable driver.

This feature would provide users with greater flexibility and control over their save files.

**Comment by Zedb0T** [comment]
This was made possible recently by @xTVaser in this PR https://github.com/open-goal/jak-project/pull/3477
